framer/Layer.coffee

```diff
@@ -1017,17 +985,18 @@ class exports.Layer extends BaseClass
 		enumerable: false
 		exportable: false
 		importable: false
-		get: -> @_stateMachine.states
+		get: ->
+			@_states ?= new LayerStates(@)
+			return @_states
```

Comment:
Well, I'd like that, but:

1) There is no hook to see if a state gets created (because it's just an object key).
2) This guarantees backwards compat